### ISSUE TITLE
BetterJoy causes system to hang or freeze, stops CEMU from loading, cannot open task manager.

### ISSUE THREAD

**Issue by particleastro** [Davidobot/BetterJoyForCemu#227]
Hi, First thanks for the effort and sorry my first post is an issue.
I have a fresh install of Cemu 1.16, with Cemuhook.  I have had Legend of Zelda running fine with the keyboard set up.

I can connect my pro switch controller to my PC, and the device is found and reported as installed, and I can run the windows configuration to see all the buttons axes etc working fine.

I have extracted betterjoy v5 and v6 in its own folder and in the Cemu folder with the same results.

I extract then run as the administrator the driver install.  I can see the HID service running in task manager.  Then the issues begin.

Sometimes if I then open betterjoy with task manager open the system will completely freeze, requiring a hard reset.  Sometimes I open betterjoy and the icon appears in the tray (in the case of v6) but nothing else happens, right or left clicking doesnt open anything.  With v5 50% of the time nothing happens.  If I open task manager to close the program, freeze then hard reset.

A few times the betterjoy screen half loaded, couldnt be closed, then same as before with the freeze.

Once I got betterjoy to connect to the pro controller, said all systems go, but then cemu wouldn't open, i just got the little blue loading circle.  When I tried to close betterjoy, system freeze, hard reset.

Once I got betterjoy to connect to the controller, open cemu, but there were no controllers detected in cemu, then freeze, then reset.

I have tried just opening betterjoy on its own, sometimes it will, sometimes the icon will appear in the tray, sometimes i immediately get a freeze.  I have tried uninstalling the drivers and clearing the black list, everything occurs the same. 

Been at this five hours now, any help gratefully appreciated.

**Comment by particleastro** [comment]
OK so spent sme more time on this this morning.

On my Win 7 Ultimate laptop, exactly the same as above.  i uniinstalled the laptops Railtek bluetooth driver and installed the toshiba blue tooth stack as I have read that is more reliable.  betterjoy would experience a .NET unhandled exception.

I uninstalled the toshiba stack, better joy runs fine.  I connected the pro controller to the laptop via USB, betterjoy found it then immediately after the laptop froze, forcing a hard reset.

So to sumarise I have tried connecting via usb and bluetooth, using windows, railtek and toshiba drivers, and the same thing happens each time.  I also have access to a windows 8.1 laptop, which I am currently updating to see what happens here.
